contracts/interfaces/igloomaster.sol

```diff
@@ -0,0 +1,1458 @@
+// SPDX-License-Identifier: MIT
+pragma solidity =0.8.6;
```

Comment:
Is this right? Most of our codebase is 0.6.X and this is only an interface after all